src/content/post/2024-07-31-kuzu-v-0.5.0.md

```diff
@@ -136,22 +135,74 @@ CREATE NODE TABLE Person (id BIGINT, name STRING, age BIGINT, PRIMARY KEY(id));
 
 #### Create sequence
 
-#### Create/Drop table if not 
+On top of `SERIAL` data type, we add `SEQUENCE` whose range and step can be cutomized in the create statement. The following example shows how to create, read and update a sequence
+```cypher
+CREATE SEQUENCE Seq INCREMENT 1 MINVALUE 1 NO MAXVALUE START 1;
+RETURN nextval('Seq');
+RETURN currval('Seq');
+```
 
-### Progress bar in CLI and Kùzu Explorers
+#### Create table if not exists
+By default, Kùzu throw an exception when trying to create a table whose name already exists in the database. User can use `CREATE ... IF NOT EXISTS` to avoid confliction. E.g.
+```cypher
+CREATE NODE TABLE IF NOT EXISTS UW(ID INT64, PRIMARY KEY(ID));
+```
 
-### C-API improvements
-Version 0.5.0 of Kùzu introduces several enhancements to the C APIs. First, we have replaced return values with out parameters, simplifying object reuse. Second, functions that may fail now return a kuzu_state value, streamlining error handling. Lastly, we have added several utility functions for data types such as `kuzu_date_t`, `kuzu_timestamp_t`, and `kuzu_interval_t`, enhancing the API's usability.
+#### Drop table if exists
+Similar to the above case, user can drop table only if it exists. E.g.
+```cypher
+DROP TABLE IF EXISTS UW;
+```
+
+#### List lambda function
+We added three list lambda functions
+```cypher
+RETRUN list_transform([1,2,3], x->x+1);
+RETURN list_filter([1,2,3], x->x>1);
+RETURN list_reduce([1,2,3], (x,y)->x+y) 
+``` 
+
+#### Progress bar in CLI and Kùzu Explorers
+For long running queries, user can enable progress bar through
+```
+CALL progress_bar=true;
+```
+
+Progress bar is currently available in CLI and explorer.
+
+#### C-API improvements
+We made several improvements over C APIs in this release.
+
+- Replace return values with out parameters, simplifying object reuse.
+- Make functions that may fail now return a kuzu_state value, streamlining error handling. 
+- Add data type utility functions such as `kuzu_date_t`, `kuzu_timestamp_t`, and `kuzu_interval_t`.
 
 ## Performance improvements
 
 ### MVCC
 
 Version 0.5.0 of Kùzu brings numerous updates and improvements to the multi-version concurrency control (MVCC) protocol.
 
-#### INT128 compression
+### Hint join order
+Picking an optimal join order can be challenging when it comes to a large query graph. We introduce join order hints to by-pass optimizer and enforce a specific join order. E.g.
```

Comment:
@andyfengHKU could you add a sentence after the code snippet below explaining the benefit? What performance improvements did we notice? etc.

Reply:
Done. Also moved this to feature section instead of performance improvement section.